tests/container_disk_test.go

```diff
@@ -174,22 +146,17 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting multiple VMIs", func() {
 		Context("with ephemeral registry disk", func() {
 			It("[test_id:1465]should success", func() {
-				num := 5
-				vmis := make([]*v1.VirtualMachineInstance, 0, num)
-				objs := make([]runtime.Object, 0, num)
-				for i := 0; i < num; i++ {
-					vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
-					// FIXME if we give too much ram, the vmis really boot and eat all our memory (cache?)
-					vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1M")
-					obj := tests.RunVMI(vmi, 10)
+				const count = 5
+				var vmis []*v1.VirtualMachineInstance
+				for i := 0; i < count; i++ {
+					vmi, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), libvmi.NewCirros())
+					Expect(err).ToNot(HaveOccurred())
 					vmis = append(vmis, vmi)
-					objs = append(objs, obj)
-				}
-
-				for idx, vmi := range vmis {
-					verifyContainerDiskVMI(vmi, objs[idx])
+					for _, vmi := range vmis {
+						verifyContainerDiskVMI(virtClient, vmi)
+					}
```

Comment:
Honestly, I don't undestand why we should verify all the vmi at every iteration (O(n<sup>2</sup>)) instead of create 2 `for` cycle (O(n))

Reply:
I don't understand either what is the point here, but I took that from the original test. IMO it should suffice to create all the VMs and then verify them in a second loop, as you said. If five VMIs were verified, less VMIs should be OK too.